packages/forms-web-app/src/validators/shared/index.js

```diff
@@ -0,0 +1,17 @@
+const { body } = require('express-validator');
```

Comment:
File name is `index.js`, do we want this? Maybe something like `not-empty-and-length.js`?

Reply:
from a compiler perspective it wouldn't make much difference... I will create both files and export them from the index then.